src/Http/Http.Results/src/EmptyHttpResult.cs

```diff
@@ -7,14 +7,18 @@ namespace Microsoft.AspNetCore.Http;
 /// Represents an <see cref="IResult"/> that when executed will
 /// do nothing.
 /// </summary>
-internal sealed class EmptyResult : IResult
+public sealed class EmptyHttpResult : IResult
 {
-    internal static readonly EmptyResult Instance = new();
-
-    private EmptyResult()
+    private EmptyHttpResult()
     {
     }
 
+    /// <summary>
+    /// Gets an instance of <see cref="EmptyHttpResult"/>.
+    /// </summary>
+    public static EmptyHttpResult Instance => new();
```

Comment:
As is this would allocate every time it's accessed.
```suggestion
    public static EmptyHttpResult Instance { get; } = new();
```